src/puzzle.rs

```diff
@@ -0,0 +1,339 @@
+//! Implements a crossword puzzle container
```

Comment:
I have not modified the contents of this file.  Not sure it there was a better way to make git(hub) realize that.

Reply:
Since writing this, I have  deleted a commented out constructor in `puzzle.rs`.